src/core/annotation.js

```diff
@@ -45,8 +45,8 @@ class AnnotationFactory {
    * @param {Object} ref
    * @param {PDFManager} pdfManager
    * @param {Object} idFactory
-   * @returns {Promise} A promise that is resolved with an {Annotation}
-   *   instance.
+   * @returns {Promise<Annotation>} A promise that is resolved with an
```

Comment:
As mentioned *multiple* times in PR #10575, no files in the `src/core/` directory should be included in this PR since that code isn't actually accessible through the API (given that it runs in a worker-thread).

Reply:
reverted